src/Loop.php

```diff
@@ -275,6 +275,34 @@ public static function setErrorHandler(callable $callback = null)
     }
 
     /**
+     * Retrieve an associative array of information about the event loop driver.
+     *
+     * The returned array MUST contain the following data describing the driver's
+     * currently registered watchers:
+     *
+     *  [
+     *      "defer"         => ["enabled" => int, "disabled" => int],
+     *      "delay"         => ["enabled" => int, "disabled" => int],
+     *      "repeat"        => ["enabled" => int, "disabled" => int],
+     *      "on_readable"   => ["enabled" => int, "disabled" => int],
+     *      "on_writable"   => ["enabled" => int, "disabled" => int],
+     *      "on_signal"     => ["enabled" => int, "disabled" => int],
+     *      "keep_alive"    => int,
+     *  ];
+     *
+     * keep_alive refers to the number of referenced watchers.
```

Comment:
`"watchers" => ["referenced" => int, "unreferenced" => int]`? I think I like that.